Makefile

```diff
@@ -27,7 +27,7 @@ fmt: install
 	@$(ESLINT) --fix .
 .PHONY: fmt
 
-test: install
+test: install lint
```

Comment:
Nathan removed this when we did a mass update. Most things didn't pass new linting rules, so we disabled linting on everything so we weren't blocked on rolling everything out.